io.c

```diff
@@ -8943,6 +8943,12 @@ rb_io_make_open_file(VALUE obj)
  *    fd = IO.sysopen(path) # => 3
  *    IO.new(fd)            # => #<IO:fd 3>
  *
+ *  The new \IO object does not inherit encoding:
```

Comment:
It may be worth it to explain that file descriptors (integers) do not have encoding, so there is no encoding to inherit.  The current wording makes it sound like inheritance is possible but not done.

Reply:
Done.